app/ai/text/blueprint/agent.py

```
@@ -0,0 +1,514 @@
+            start_time,
+        )
+        return
+
+    # Stage 2: Stream events from the deep agent
+    yield _make_status(
+        PipelineStage.PLANNING,
+        "Planning template generation steps...",
+        start_time,
+    )
+
+    PipelineStage.PLANNING
```

Ruff (0.15.2)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)